packages/neos-ui/src/Containers/ErrorBoundary/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {FallbackProps, ErrorBoundary as ReactErrorBoundary} from 'react-error-boundary';
+import styles from './style.module.css';
+import { Logo } from '@neos-project/react-ui-components';
+
+const ErrorFallback = (props: FallbackProps) => {
+    return (
+        <div className={styles.container}>
+            <div>
+                <Logo />
+                <h1 className={styles.title}>Sorry, but the Neos UI could not recover from this Error.</h1>
+                <p>Please reload the application, or contact your system administrator with the given details.</p>
+                {props.error instanceof Error &&
+                    <>
+                        <p>
+                            Name: {props.error.name || '-'}
+                        </p>
+                        <p>
+                            Message: {props.error.message || '-'}
+                        </p>
+                        <p>Stacktrace:</p>
+                        <pre className={styles.stackTrace}>{props.error.stack}</pre>
+                    </>
+                }
+                <p>For more information about the Error please refer to the JavaScript console.</p>
+            </div>
```

Comment:
A reload button would also be nice, so users can use it to continue.

Reply:
agreed and added ;)